src/SIL.XForge/Services/UserService.cs

```diff
@@ -109,6 +113,20 @@ await using (IConnection conn = await _realtimeService.ConnectAsync(curUserId))
                         update => update.Set(put => put.ParatextTokens, newPTTokens)
                     );
                 }
+                else if (newPTTokens.IssuedAt < userSecret.ParatextTokens.IssuedAt)
+                {
+                    string incomingIAt = newPTTokens.IssuedAt.ToString(
+                        "o",
+                        System.Globalization.CultureInfo.InvariantCulture
+                    );
+                    string currentIAt = userSecret.ParatextTokens.IssuedAt.ToString(
+                        "o",
+                        System.Globalization.CultureInfo.InvariantCulture
+                    );
+                    _logger.LogWarning(
+                        $"When updating user from auth0 profile, ignoring incoming tokens which were issued at {incomingIAt}, which is earlier than the current tokens {currentIAt}."
```

Comment:
Would we want to log the user Id so we can begin diagnosis?

Reply:
Yes :). Thank you.